pkg/diff/sql_generator.go

```diff
@@ -432,7 +465,27 @@ func (schemaSQLGenerator) Alter(diff schemaDiff) ([]Statement, error) {
 		return nil, fmt.Errorf("unioning table and trigger graphs: %w", err)
 	}
 
-	return tableGraphs.toOrderedStatements()
+	statements, err := tableGraphs.toOrderedStatements()
+	if err != nil {
+		return nil, fmt.Errorf("getting ordered statements from tableGraph: %w", err)
+	}
+
+	return mergeStatementsInOrder(
```

Comment:
Maybe add a comment explaining why you're enabling extensions first and disabling them last, so other folks have context on the problem?